### PR TITLE
CONTRIBUTING.md File

### LINKED ISSUE #11: Creating CONTRIBUTING.md
Here is the guideline base on lecture notes:

It is a good practice to include information about how others, outside the core team, can contribute to your project somewhere in your repository. This is typically done as a separate file in the repository called CONTRIBUTING.md. Here are some examples of this file:

[Altair](https://github.com/altair-viz/altair/blob/master/CONTRIBUTING.md)
[dplyr](https://github.com/tidyverse/dplyr/blob/master/.github/CONTRIBUTING.md)
[Atom editor](https://github.com/atom/atom/blob/master/CONTRIBUTING.md) (very comprehensive)
If you’re interested in this, [here’s a quick guide](https://help.github.com/en/github/building-a-strong-community/setting-guidelines-for-repository-contributors) to creating this file from GitHub, but in the meantime, you can use the following snippet as a template:

We welcome all contributions to this project!
If you notice a bug, or have a feature request,
please open up an issue [here](https://github.com/UBC-DSCI/REPOSITORY_NAME/issues).
If you'd like to contribute a feature or bug fix,
you can fork our repo and submit a pull request.
We will review pull requests within 7 days.
All contributors must abide by our [code of conduct](CODE_OF_CONDUCT.md).

### PR DESCRIPTION
**Pull request by atabak-alishiri**
It should be reviewed especially the links to our project based on the name of our project/repo.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Contributing to the Predicting English Proficiency Project
+
+We welcome all contributions to this project! Your input and support are essential in making this tool more effective and user-friendly.
+
+## Reporting Bugs and Feature Requests
+
+If you notice a bug or have a feature request, please help us by opening an issue in our GitHub repository:
+
+- **Open an Issue**: You can report bugs or suggest features [here](link).
```
Comment (farrandi) [inline CONTRIBUTING.md]: We need to replace the link with a working link.

**Review by rbouwer — APPROVED** [review]

**Review by salva-u — APPROVED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (farrandi) [reply CONTRIBUTING.md]: https://github.com/UBC-MDS/522-workflows-group-18/issues

**Review by farrandi — COMMENTED** [review]
See reviewed comment. I think we should replace "link" with the actual link I suggested

File: CONTRIBUTING.md (hunk 1 above)
Comment (atabak-alishiri) [reply CONTRIBUTING.md]: Thanks for mentioning that, I fixed the link.

**Review by farrandi — APPROVED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (rbouwer) [reply CONTRIBUTING.md]: Going to merge now

**Pull request merged**